pkg/scheduler/remotescheduler.go

```diff
@@ -106,17 +107,20 @@ func (rs *RemoteScheduler) schedule(state cluster.State) {
 		return
 	}
 
+	statusUpdater := NewClusterStatusUpdater(rs.inventoryWatch.Inventory(), state, components, rs.logger)
+	go statusUpdater.Run()
```

Comment:
Oh, sorry missed that one: we should also pass the CTX to the status-updater and check the `ctx.Done()` channel in the loop for a graceful shutdown of the goroutine